experimental/rules_python/wheelmaker.py

```diff
@@ -157,13 +158,38 @@ def add_metadata(self, extra_headers, description, classifiers, requires,
         metadata += "\n"
         self.add_string(self.distinfo_path('METADATA'), metadata)
 
-    def add_entry_points(self, console_scripts):
+    def add_entry_points(self, entry_points):
         """Write entry_points.txt file to the distribution."""
         # https://packaging.python.org/specifications/entry-points/
-        if not console_scripts:
+        import re
+        pattern = re.compile(r'([^;]+);([^=]+)=(.*)') # group;name=ref
```

Comment:
FYI, an earlier draft of this code required `ref` to be in the format `a:b`. Turns out that's only one of the allowed formats for an object reference, so I dropped it.